python/cudf_polars/cudf_polars/dsl/tracing.py

```diff
@@ -0,0 +1,48 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Tracing."""
+
+from __future__ import annotations
+
+import typing
+
+import nvtx
+
+if typing.TYPE_CHECKING:
+    from collections.abc import Callable
+
+    import cudf_polars.containers.dataframe
+
+
+CUDF_POLARS_NVTX_DOMAIN = "cudf_polars"
+
+
+def wrap_do_evaluate(
```

Comment:
I see "Emits a log record with the node type and some additional metadata (currently using structlog)" in the PR description. But I don't think you're logging anything anywhere. Is that still happening?